From de6d11b8587be186d089e692fc410008d4d87034 Mon Sep 17 00:00:00 2001
Message-Id: <de6d11b8587be186d089e692fc410008d4d87034.1414996573.git.chang-joon.lee@intel.com>
From: Rafael Barbalho <rafael.barbalho@intel.com>
Date: Tue, 28 Oct 2014 17:05:24 +0000
Subject: [PATCH 01/10] FOR_UPSTREAM [VPG]: drm/i915: Fix PPGTT initialisation
 when execlist are enabled

The following commit introduced a wrong return:

commit 796f9b846b457842212ebc73ecc4c70f9f1f896c
Author: Oscar Mateo <oscar.mateo@intel.com>
Date:   Fri Aug 15 12:01:31 2014 +0100

    FROM_UPSTREAM [VPG]: drm/i915/bdw: Don't write PDP in the legacy way when using LRCs

For: GMINL-1778
Change-Id: I19ffe62d8f6f99d028acd1aaa5650a0772a3edfb
Signed-off-by: Rafael Barbalho <rafael.barbalho@intel.com>
---
 drivers/gpu/drm/i915/i915_gem_gtt.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/i915_gem_gtt.c b/drivers/gpu/drm/i915/i915_gem_gtt.c
index 4d16b1b..eb61631 100644
--- a/drivers/gpu/drm/i915/i915_gem_gtt.c
+++ b/drivers/gpu/drm/i915/i915_gem_gtt.c
@@ -894,7 +894,7 @@ static int gen8_ppgtt_enable(struct i915_hw_ppgtt *ppgtt)
 	 * and the PDPs are contained within the context itself.  We don't
 	 * need to do anything here. */
 	if (i915.enable_execlists)
-		return;
+		return 0;
 
 	for_each_ring(ring, dev_priv, j) {
 		I915_WRITE(RING_MODE_GEN7(ring),
-- 
1.7.9.5

